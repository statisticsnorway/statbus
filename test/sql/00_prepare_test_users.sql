BEGIN;

-- Insert users for other tests.

SELECT * FROM public.statbus_user_create('test.super@statbus.org', 'super_user'::statbus_role_type, 'Super#123!');
SELECT * FROM public.statbus_user_create('test.regular@statbus.org', 'regular_user'::statbus_role_type, 'Regular#123!');
SELECT * FROM public.statbus_user_create('test.restricted@statbus.org', 'restricted_user'::statbus_role_type, 'Restricted#123!');

CALL test.set_user_from_email('test.regular@statbus.org');

END;
