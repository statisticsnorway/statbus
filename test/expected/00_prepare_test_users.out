BEGIN;
-- Insert users for other tests.
SELECT * FROM public.statbus_user_create('test.super@statbus.org', 'super_user'::statbus_role_type, 'Super#123!');
         email          |  password  
------------------------+------------
 test.super@statbus.org | Super#123!
(1 row)

SELECT * FROM public.statbus_user_create('test.regular@statbus.org', 'regular_user'::statbus_role_type, 'Regular#123!');
          email           |   password   
--------------------------+--------------
 test.regular@statbus.org | Regular#123!
(1 row)

SELECT * FROM public.statbus_user_create('test.restricted@statbus.org', 'restricted_user'::statbus_role_type, 'Restricted#123!');
            email            |    password     
-----------------------------+-----------------
 test.restricted@statbus.org | Restricted#123!
(1 row)

CALL test.set_user_from_email('test.regular@statbus.org');
END;
